### PR TITLE
fix make dimensions.aspectRatios key of theme.json files translatable

### LINKED ISSUE #61767: Aspect ratio preset names are not translatable
### Description

Once again nobody thinks about internationalization... 😞 

#47271 added support for aspect ratio presets via theme.json. Unfortunately those presets are not translatable.

Support needs to be added to `theme-i18n.json`, then once the feature is stable the schema needs to be updated on the CDN as well.

In the UI, the presets need to run through the translation function so that they can be correctly translated.

### Step-by-step reproduction instructions

1. Add aspect ratio presets to `theme.json`
2. Change your language
3. Wonder how you could translate the preset names

### Screenshots, screen recording, code snippet

_No response_

### Environment info

- Current Gutenberg RC

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

### PR DESCRIPTION
**Pull request by fabiankaegy**
## What?

Make Aspect Ratio preset names in `theme.json` translatable.

## Why?

Closing #61767

## How?

Adding the schema to the `theme-i18n.json` file.

### CODE REVIEW

**Comment by fabiankaegy** [comment]
@swissspidy I also updated the PR in core to include this: https://github.com/WordPress/wordpress-develop/pull/6567/commits/ddc68776cbee82d2cd80bcdc7d7814a3050d0ee2

**Comment by Mamaduka** [comment]
@fabiankaegy, @swissspidy, what are the testing instructions for `theme-i18n.json` changes?

**Comment by fabiankaegy** [comment]
@Mamaduka I am unsure myself sadly. I just followed the code examples and looked at a few other PR's that implemented it

**Comment by swissspidy** [comment]
Manually updating translation files & then verifying that the strings are transkated woukd be one option.

**Comment by t-hamano** [comment]
If I understand correctly, `theme-i18n.json` exists to make preset labels translatable, right?

I tried testing existing presets, but for some reason they don't translate.

For example, color presets are translated into Japanese on GlotPress:

https://translate.wordpress.org/projects/wp-themes/twentytwentyfour/ja/default/?filters%5Bterm%5D=Accent&filters%5Bterm_scope%5D=scope_originals&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=original&sort%5Bhow%5D=asc

image

But when I change the site language to Japanese, the color palette is not translated 🤔

image

**Review by ajlende — APPROVED** [review]
As far as I'm aware, this is the only setting that should control i18n for theme.json. I'm going to get this merged for the 18.4 release, but I think it's worthwhile looking into the issue of it not working even for the pre-existing color palette that @t-hamano shared above.

**Comment by ajlende** [comment]
I just cherry-picked this PR to the release/18.4 branch to get it included in the next release: 5d68089808

File: backport-changelog/6.6/6567.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+https://github.com/WordPress/wordpress-develop/pull/6567
+
+* https://github.com/WordPress/gutenberg/pull/47271
+* https://github.com/WordPress/gutenberg/pull/61774
```
Comment (ellatrix) [inline backport-changelog/6.6/6567.md:4]: Thank you ❤️

File: lib/theme-i18n.json [hunk 2]
```diff
@@ -38,6 +38,13 @@
 				}
 			]
 		},
+		"dimensions": {
+			"aspectRatios": [
+				{
+					"name": "Aspect ratio name"
+				}
+			]
+		},
```
Comment (ellatrix) [inline lib/theme-i18n.json:47]: This needs to be backported?

**Pull request merged**